desc/equilibrium/equilibrium.py

```diff
@@ -1975,7 +1975,7 @@ def _optimize(  # noqa: C901
                 print("Trust-Region ratio = {:9.3e}".format(tr_ratio[0]))
 
             # perturb + solve
-            (_, predicted_reduction, dc_opt, dc, c_norm, bound_hit,) = optimal_perturb(
+            (_, predicted_reduction, dc_opt, dc, c_norm, bound_hit) = optimal_perturb(
```

Comment:
this was a really unnecessary change...

Reply:
I disagree. When you have that extra comma at the end, the black formatter expands the tuple onto multiple lines with each variable on a separate line (at least mine does). So this change reduced the same code from 6 lines down to 1 line.